src/doc_builder/commands/build.py

```diff
@@ -66,6 +66,22 @@ def locate_kit_folder():
     # TODO for the future if asked for, if we can't locate the kit folder git clone doc-builder and cache it somewhere.
 
 
+def get_default_branch_name(repo_folder):
+    try:
+        p = subprocess.run(
+            ["git", "branch"],
+            stderr=subprocess.PIPE,
+            stdout=subprocess.PIPE,
+            check=True,
+            encoding="utf-8",
+            cwd=repo_folder,
+        )
+        branches = p.stdout.strip()
+        return "master" if "master" in branches else "main"
+    except Exception:
+        return "main"
```

Comment:
why is main returned on exception?

Reply:
If you want to do it perfectly you could parse it from the result of `git symbolic-ref refs/remotes/origin/HEAD`, but it's fine for me to take `main` as the default.